lib/files.js

```diff
@@ -122,6 +122,11 @@ module.exports.isDirectory = (file) => {
   return getStats(file).isDirectory();
 };
 
+module.exports.isVisibleFile = (path) => {
+  const pathParts = path.split('/');
```

Comment:
You should be able to use 'splitPath' in the files module, which uses a platform specific separator.